src/status_im/ui/screens/communities/community.cljs

```diff
@@ -204,8 +204,8 @@
         :header                       [categories-accordion community-id chats categories edit {:from-chat from-chat}]
         :footer                       [rn/view {:height 68}]}])))
 
-(defn channel-preview-item [{:keys [id color name]}]
-  (let [color (or color colors/default-community-color)]
+(defn channel-preview-item [{:keys [id name]}] ;; Removed color field because it was displaying channel colors without joining
```

Comment:
im not sure if we need such commits in the code, in github its ok, but in code nobody knows whats is color field because there is no this field anymore ;)